### PR TITLE
style-theme

### PR DESCRIPTION
**Pull request by EFT-Defra**
This pull request restyles the indicators-by-theme page. It visually breakups up the indicators similarly to the individual goal pages.

It also re-writes the if statements that set the status description as a case statement.

The page should now look like this:
image

### CODE REVIEW

File: _themes/index.html [hunk 1]
```diff
@@ -9,59 +9,66 @@ <h1>
...
+                    {% when "notstarted" %}
+                        {% assign status_desc = page.t.status.exploring_data_sources %}
+
+                    {% when "inprogress" %}
+                        {% assign status_desc = page.t.status.statistics_in_progress %}
+
+                    {% when "complete" %}
+                        {% assign status_desc = page.t.status.reported_online %}
+
+                {% endcase %}
```
Comment (EFT-Defra) [inline _themes/index.html:39]: Here I'm rewriting a series of IF statements as a CASE statement. It does the same thing but, I think, it's a little cleaner.

File: _themes/index.html [hunk 2]
```diff
@@ -9,59 +9,66 @@ <h1>
...
     <div class="container">
         {% for theme in site.themes %}
-        <h2>
-            {{ theme.name }}
-        </h2>
-        {% assign theme_indicators = page.indicators | where_exp: "indicator", "indicator.indicator_number contains theme.code"  %}
+            <h2>
+                {{ theme.name }}
+            </h2>
+            <br>
```
Comment (EFT-Defra) [inline _themes/index.html:18]: This adds a line break between the theme name and theme indicators.

File: _themes/index.html [hunk 3]
```diff
@@ -9,59 +9,66 @@ <h1>
...
+                                </a>
+                            </span>
+                        </div>
+                        <div class="col-xs-2">
+                            <span class="status {{ status_css }}">
+                                <span class="sr-only">
+                                    {{ page.t.indicator.status }}
+                                </span>
+                                {{ status_desc }}
                             </span>
```
Comment (EFT-Defra) [inline _themes/index.html:64]: The page uses the [Bootstrap Grid system](https://getbootstrap.com/docs/4.0/layout/grid/), which breaks row up into 12 columns. The class="col-xs-*" bits specify the sizes of the columns in this row: 1, 9, 2. The "xs" bit means these columns are formatted for mobile screens size upwards.

File: _themes/index.html [hunk 4]
```diff
@@ -9,59 +9,66 @@ <h1>
...
-                            {{ page.t.indicator.status }}
-                        </span>
-                        <span class="status {{ status_css }}">
-                            {{ status_desc }}
-                        </span>
+                        </div>
                     </div>
                 </div>
-            </div>
+                <hr>
```
Comment (EFT-Defra) [inline _themes/index.html:68]: This adds the faint grey line under each indicator.

**Comment by TimAshelford** [comment]
Thanks, all seems clear and logical

**Review by TimAshelford — APPROVED** [review]

**Pull request merged**